### DISCUSSION TITLE
Github Support is taking too long.

### DISCUSSION THREAD

**Discussion by GameVerseMaster** [community/community#121492] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Its been a month since I opened another ticket, 4 months since I sent in my original ticket by email. My main account was hacked, and the phone number I own I believe is still connected to it. However the email and password associated with the account has been changed, I desperately need access to my account. Please github check my ticket! (#2705814) Im wondering if I could use the phone number as verification, to regain access to my account.

What gives?

**Comment by ghostinhershell** [comment]
Hi @GameVerseMaster,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by GameVerseMaster** [comment]
Hi, it has been another 2 weeks, and I still have not gotten a response on either of my tickets.
